packages/core/src/messagingProtocols/mls/E2EIdentityService/E2EIServiceInternal.ts

```diff
@@ -35,86 +34,50 @@ import {createNewOrder, finalizeOrder} from './Steps/Order';
 import {E2EIStorage} from './Storage/E2EIStorage';
 import {AuthData, InitialData} from './Storage/E2EIStorage.schema';
 
-import {NewCrlDistributionPointsPayload} from '../MLSService/MLSService.types';
-
 export class E2EIServiceInternal {
-  private static instance: E2EIServiceInternal;
   private readonly logger = logdown('@wireapp/core/E2EIdentityServiceInternal');
-  private _acmeService?: AcmeService;
-  private _initialData?: InitialData;
+  private acmeService: AcmeService;
 
-  private constructor(
+  public constructor(
     private readonly coreCryptoClient: CoreCrypto,
     private readonly apiClient: APIClient,
-    private readonly e2eiServiceExternal: E2EIServiceExternal,
     /** number of seconds the certificate should be valid */
     private readonly certificateTtl: number,
     private readonly keyPackagesAmount: number,
-    private readonly dispatchNewCrlDistributionPoints: (payload: NewCrlDistributionPointsPayload) => void,
-  ) {}
-
-  // ############ Public Functions ############
-
-  public static async getInstance(params?: InitParams): Promise<E2EIServiceInternal> {
```

Comment:
no more singleton features to this class, it's standard now. 
This was not needed since this class is only instantiated once.